docs/mkdocs.yml

```diff
@@ -720,6 +720,8 @@ markdown_extensions:
 extra:
   social:
     - icon: fontawesome/brands/github
+      link: 'https://github.com/pnp/cli-microsoft365'
+    - icon: fontawesome/brands/twitter
       link: 'https://twitter.com/m365pnp'
```

Comment:
Since we have our own Twitter account, shall we point to it here rather than to the generic PnP account?